axiomapy/axiomaapi/clienteventbus.py

```diff
@@ -0,0 +1,105 @@
+"""
+Copyright © 2022 Qontigo GmbH.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+
+"""
+
+import logging
+from axiomapy.session import AxiomaSession
+from axiomapy.utils import odata_params
+from axiomapy.odatahelpers import oDataFilterHelper as od
+
+_logger = logging.getLogger(__name__)
+_logger.addHandler(logging.NullHandler())
+
+
+class ClientEventBusAPI:
+    """Access to Axioma Client Event Bus endpoints using the active session
+    """
+
+    @staticmethod
+    def get_events(return_response: bool = True):
```

Comment:
A general suggestion given this is a new class: should we add a return type as well?

Reply:
I am not sure if i understand this right. The return type is always json.
My understanding is it would be good to keep functions in sync with other functionalities. Any specific reason why you requested for this?